src/pages/home/report/ReportActionItemFragment.js

```diff
@@ -104,20 +104,22 @@ const ReportActionItemFragment = (props) => {
                         html={`<comment>${props.fragment.html + (props.fragment.isEdited ? '<edited></edited>' : '')}</comment>`}
                     />
                 ) : (
-                    <Text
-                        selectable={!canUseTouchScreen() || !props.isSmallScreenWidth}
-                        style={EmojiUtils.containsOnlyEmojis(props.fragment.text) ? styles.onlyEmojisText : undefined}
-                    >
-                        {Str.htmlDecode(props.fragment.text)}
+                    <View style={[styles.flexRow, styles.alignItemsBaseline]}>
```

Comment:
@Karim-30 unfortunately, `justifyContent: baseline` doesn't work well for a multiline message

<img width="398" alt="image" src="https://user-images.githubusercontent.com/29673073/178595125-7fd56081-d163-4992-8b55-23361a3cced4.png">

paste this message and edit it - "pen app on webPost(if not) some code block message with long lines so that it overflows.Try to move two finger (scroll) not precisely horizontally but at some other angle eg. 45Expected Result:
We have to precisely scroll to left or right or else the page will also scroll vertically."

Reply:
updated,  here how it looks like : 

web :


https://user-images.githubusercontent.com/108357004/178799584-68da6ea0-0b5a-45a0-bd88-b63a8d0b5cd0.mov


mweb :

https://user-images.githubusercontent.com/108357004/178799991-9dcb4f21-450a-462c-b0a3-e72bdb2c6aae.mov



android :


https://user-images.githubusercontent.com/108357004/178799657-b91cb114-d65d-4190-ad5b-f3e59eaeb430.mov


ios :


https://user-images.githubusercontent.com/108357004/178799784-4fd1514c-cb79-41a6-9fb6-5d571943561b.mov


desktop : 

https://user-images.githubusercontent.com/108357004/178799927-131c6304-af77-4d88-9fa2-82fe304a23bf.mov

Reply:
@Karim-30 code keeps changing, so it isn't necessary to add _updated_ screenshots :)